myapp/DataFetching.py

```diff
@@ -0,0 +1,48 @@
+import requests
+import sqlite3
+import sys
+
+
+def connect():
+    database = sqlite3.connect('testDatabase.db',isolation_level=None,timeout=10)
+    return database
+
+def fetch_stock(symbol):
+    url = "https://sandbox.iexapis.com/stable/stock/"+str(symbol)+"/quote?token=Tpk_087a52e2547e449e9cee8748f00491eb"
+    response=requests.get(url)
+    if (response.status_code)==404:
+        return 0
+    else:
+        dataset = response.json()
+    return (dataset)
+
+def is_cached(ssymbol,database):
+    c1 = database.cursor()
+    res = list(c1.execute('''select symbol from stocks '''))
+    res=flatter(res)
+    if ssymbol in res:
+        return True
+    else:
+        return False
+
+def add_stock_to_db(symbol,database):
+    if is_cached(symbol.upper(),database) is True:
+
+        sys.exit()
+    else:
+        stock_data=fetch_stock(symbol)
+        if stock_data==0:
+            exit(0)
+        else :
+            c1 = database.cursor()
+            c1.execute("INSERT INTO stocks VALUES (?,?)", [stock_data["symbol"], stock_data["companyName"]])
+
+def flatter(tu):
+    l = []
```

Comment:
> This function is used to reformat the Results coming from the SELECT query, as you can see without this function the result looks as follow: 

<img width="890" alt="Screen Shot 2019-10-18 at 17 40 40" src="https://user-images.githubusercontent.com/48990849/67103742-644f8200-f1ce-11e9-8db4-bd477e205fb3.png">

>And when applying the function it looks as follows, it will be easier to use and modify .. etc.

<img width="624" alt="Screen Shot 2019-10-18 at 17 42 16" src="https://user-images.githubusercontent.com/48990849/67103860-9cef5b80-f1ce-11e9-9b11-feea917a7d5c.png">

Reply:
oh yes, sorry.. i was wrong with my implementation. 
in general i would do it like this then:
def reformat_select_query(self, query_res ):
    return [ row[0] for row in query_res  ]